test/Curator/SnapshotSpec.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Curator.SnapshotSpec ( spec ) where
+
+import Curator.Snapshot
+import qualified Data.Map as M
+import qualified Data.Set as S
+import Data.Text (unpack)
+import Distribution.Types.PackageName (mkPackageName)
+import Distribution.Types.VersionRange (noVersion)
+import Test.Hspec
+
+spec :: Spec
+spec = do
+  describe "pkgBoundsError" $
+    context "when there is more than one github pin" $
```

Comment:
Was this supposed to say "ping" instead of "pin"?

Reply:
Not really. Actually, I'm referring to github usernames that could be in the list of maintainers.

Reply:
I can change that, no problem. Sorry just realized that was a typo.